spec/features/admin_user/nav_access_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "rails_helper"
+
+RSpec.describe "As a admin level user" do
```

Comment:
It looks like these test files are organized differently than our tests, but there shouldn't be any problems